frontend/src/pages/admin/users/ManageUsers.tsx

```diff
@@ -12,55 +11,101 @@ import './ManageUsers.scss';
 import { UsersList, TableStateChangeFn } from './components/UsersList';
 import { IUser } from 'interfaces';
 import { MUIDataTableState } from 'mui-datatables';
-import { IUserRecord, AccountActive } from './interfaces/IUserRecord';
+import { IUserRecord, AccountActive, accountActiveToBool } from './interfaces/IUserRecord';
+import { IUsersState, MAX_USERS_PER_PAGE } from 'reducers/usersReducer';
+import { setUsersFilter, setUsersPageSize, setUsersSort, IUsersSort } from 'actions/adminActions';
+import { sortBy } from 'lodash';
 
 const ManageUsers = () => {
   const dispatch = useDispatch();
-  const MAX_USERS_PER_PAGE = 10;
   const pagedUsers = useSelector<RootState, IPagedItems<IUser>>(state => {
-    return (state.users as IStoreUsersAction).pagedUsers;
+    return (state.users as IUsersState).pagedUsers;
   });
+
+  const pageSize = useSelector<RootState, number>(state => {
```

Comment:
I'm not sure if there is any benefit to this suggestion, but it seems like you could make a single selector for the top level state.users object and then destructure that into the values that you need, rather then making two selectors here.

Reply:
If you do that it means that even when the other properties change then the reaction to the pageSize change will be triggered even if the pageSize did not change